ui/app/styles/utils/_colors.scss

```diff
@@ -1,17 +1,22 @@
+@use 'tokens';
 // Colors that will eventually be in Structure package
 
+// Design System Colors: https://design-system-components-hashicorp.vercel.app/foundations/tokens
+$token-black: var(--token-color-vault-brand-alt); // black #000000
```

Comment:
Thoughts on prefixing any design system tokens, not previously defined, with `$token-` ? or would we like it to be more specific `$ds-token-` 

Open to any other ideas!